package.json

```diff
@@ -64,6 +64,7 @@
     "lint": "npm run --silent lint:js && npm run --silent lint:jenkinsfile && npm run --silent lint:package-names",
     "lint:ci": "npm run --silent lint:js:ci && npm run --silent lint:jenkinsfile && npm run --silent lint:package-names",
     "lint:commitmsg": "commitlint -e $GIT_PARAMS",
+    "lint:errors": "npm run lint | grep --color -e '\\serror\\s' || true",
     "lint:eslint": "eslint --ignore-pattern 'packages/node_modules/generator-ciscospark/generators/*/templates/**' --ignore-pattern 'packages/node_modules/**/umd/**' --ignore-pattern 'docs/**' --ignore-path .gitignore",
     "lint:eslint:ci-reporter": "bash -c '[[ -n \"${JENKINS}\" ]]' && echo \"-f checkstyle -o reports/style/eslint.xml\" || echo ''",
     "lint:jenkinsfile": "node --eval \"fs.readFileSync('Jenkinsfile').includes('.status') ? console.log('Found \".status\" in Jenkinsfile. You meant \".result\"') && process.exit(1) : process.exit(0)\"",
```

Comment:
This doesn't emit a string `error` and would get filtered out even though it is a real error. (sidenote: this seems like a very specialized command and is it even needed any more?)